src/state/models/content/post-thread-item.ts

```diff
@@ -61,8 +61,9 @@ export class PostThreadItemModel {
     }
     return this.post.uri
   }
+
   get isThreadMuted() {
-    return this.rootStore.mutedThreads.uris.has(this.rootUri)
+    return this.data.isThreadMuted
   }
```

Comment:
The logic looks the same, so I assume this is some kind of reference/synchronization issue.

Reply:
Yeah this may indicate a deeper issue that we need to look into, because I can't point to the exact cause